### DISCUSSION TITLE
New version ; old plots

### DISCUSSION THREAD

**Discussion by fuhsirotto** [Chia-Network/chia-blockchain#2587] [Q&A]
Hello guys , how do i change my plot made in old version to the new version ? 

Im in version 1.0.1 im afraid to download the new version and lose all my plots :(

**Comment by fuhsirotto** [comment]
I found here in some page before

Click ... (3 dots) near from the add plots and there is ;))

**Comment by hoffmang9** [comment]
Plots don't care about which version of the software they are used with since Beta 8 last July 2020.
